acceptance/lib/helper.rb

```diff
@@ -1,4 +1,6 @@
 require 'beaker/dsl/install_utils'
+require 'puppetserver/acceptance/compat_utils'
+require 'puppetserver/acceptance/gem_utils'
```

Comment:
I think we need to add another directory to the load path; shouldn't have to mess with the helper file at all.